common/src/main/java/com/wynntils/mc/event/PlayerInteractEvent.java

```diff
@@ -7,29 +7,24 @@
 import com.google.common.base.Preconditions;
 import javax.annotation.Nonnull;
 import net.minecraft.core.BlockPos;
-import net.minecraft.core.Direction;
 import net.minecraft.world.InteractionHand;
 import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.Entity;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.phys.BlockHitResult;
+import net.minecraft.world.phys.EntityHitResult;
 import net.minecraftforge.eventbus.api.Cancelable;
 import net.minecraftforge.eventbus.api.Event;
 
 public class PlayerInteractEvent extends PlayerEvent {
     private final InteractionHand hand;
-    private final BlockPos pos;
-
-    private final Direction face;
-
     private InteractionResult cancellationResult = InteractionResult.PASS;
 
-    private PlayerInteractEvent(Player player, InteractionHand hand, BlockPos pos, Direction face) {
+    private PlayerInteractEvent(Player player, InteractionHand hand) {
         super(Preconditions.checkNotNull(player, "Null player in PlayerInteractEvent!"));
         this.hand = Preconditions.checkNotNull(hand, "Null hand in PlayerInteractEvent!");
```

Comment:
I never noticed these before. Why do we not trust vanilla to have non-null values?

Reply:
I think this is forge event code from a few months ago.

Reply:
anyways i removed this and the annotations